### DISCUSSION TITLE
Memory swap and object dispose in directus

### DISCUSSION THREAD

**Discussion by moayadsquad** [directus/directus#13319] [Archive]
Does directus handle memory swap and object dispose? 
my mysql server database have like 500K records

**Comment by br41nslug** [comment]
Swap is managed by your operating system when you run out of memory.
I suspect the mysql daemon is running out of memory for some complex query.

**Reply by moayadsquad** [reply]
The issue happen when directus server I/O weight get around 70 - 60, I don't think issue related to mysql server, the server a few users on it
